### PR TITLE
add gp-winner instructions

### PR DESCRIPTION
**Pull request by bruniss**
just added to bottom, probably getting low click-through to kaggle data and most of it is out of date at this point, not that it is not relevant, just that fragment data doesnt really transfer to scroll1 and likely doesnt to the others.

### CODE REVIEW

File: docs/07_tutorial4.md [hunk 1]
```diff
@@ -155,3 +155,96 @@ So how can a machine learning model detect ink? In the electron microscope image
 Now let’s create a model! This part of the tutorial is over [on Kaggle as a notebook](https://www.kaggle.com/code/jpposma/vesuvius-challenge-ink-detection-tutorial).
 
 To run more advanced models on the [scroll segments](data_segments), check out the winning code from the [First Letters Prize](firstletters) and the [2023 Grand Prize](grandprize).
+
```
Comment (danieldegroot2) [inline docs/07_tutorial4.md:158]:
```suggestion

### Running the 2023 Grand Prize model

```
Could use a heading, similar to existing tutorials.

File: docs/07_tutorial4.md [hunk 2]
```diff
@@ -155,3 +155,96 @@ So how can a machine learning model detect ink? In the electron microscope image
...
+    start_idx:int=15 
+    workers: int = 4 #number of threads to use, increase or decrease depending on your specs, not to exceed number of threads avail
+    batch_size: int = 512 #increase or decrease depending on hardware, would have some difficulty above 512 on regular consumer hardware. if you get CUDA: out of memoery errors, lower this or increase stride, or both
+    size:int=64 #window size
+    reverse:int=0 #use to reverse the order of the layers during inference
+    device:str='cuda' #change to cpu or mps if on device with no gpu or apple device, respectively
+```
+these all can be passed as arguments. ex: `python inference_timesformer.py --segment_id 20231210121321 20231221180251 --segment_path $(pwd)/train_scrolls --model_path timesformer_wild15_20230702185753_0_fr_i3depoch=12.ckpt`
+
+advanced: to change the number of layers the model runs inference on, you must change the following:
```
Comment (danieldegroot2) [inline docs/07_tutorial4.md:219]:
```suggestion
Advanced: to change the number of layers the model runs inference on, you must change the following:
```

File: docs/07_tutorial4.md [hunk 3]
```diff
@@ -155,3 +155,96 @@ So how can a machine learning model detect ink? In the electron microscope image
...
+
+```bash
+
+    in the read_image_mask function, where it is defined
```
Comment (danieldegroot2) [inline docs/07_tutorial4.md:223]: (I will assume these seperated lines are intended to and will be shown as comments even without a hashtag or similar.)

**Comment by bruniss** [comment]
hey daniel, thank you so much for the improvements here, but im a having a bit of trouble. i'd like to just submit this as 1 pull request with all the changes, i think you said that the changes here are also in the other one? sorry i've not really used pull requests before, so its all new to me

**Comment by bruniss** [comment]
duplicate.redundant

**Comment by danieldegroot2** [comment]
Hey Bruniss, yes this is possible. Just wait for the old PR to be merged before modifying your main branch.
After it is merged, I find the easiest way is to just delete the repo and fork the scrollprize repo again.

You can edit the title at the top of #17.
~~Then, come back here and below, under the comment box, use the button to Close with/without comment.~~
(which you have already done.)

I have moved the suggestions back to the other PR.

**Pull request closed without merging**